content/posts/2025-08-0-8-second-webinar.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Second Ecoacoustics Webinar: BirdNET Tips and Tricks"
+date: 2025-08-08T12:00:00+10:00
+image: "/images/kookaburra.png"
```

Comment:
I'm assuming that we're purposely using the same image as the previous webinar post, and all blog posts from this series will have the same image so readers can scan what posts are a part of this series?